app/services/facebook_service.rb

```diff
@@ -0,0 +1,24 @@
+class FacebookService
+
+  def self.send_post_to_page(resource_id:, resource_type:)
+    resource = resource_type.constantize.find_by(id: resource_id)
+    return unless resource
+
+    setting = resource.settings
+    return false if setting.blank?
```

Comment:
was is der vorteil hier `false` zu returnen statt nichts, wie im fall zeile 5 z.b.?

Reply:
keiner